docs/Model/BatchUpsertRecordsRequest.md

```diff
@@ -2,10 +2,10 @@
 
 ## Properties
 
-| Name          | Type                                                                                        | Description                                                                                        | Notes      |
-| ------------- | ------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- | ---------- |
-| **pipeline**  | [**\Sajari\Model\BatchUpsertRecordsRequestPipeline**](BatchUpsertRecordsRequestPipeline.md) |                                                                                                    | [optional] |
-| **records**   | **object[]**                                                                                | A list of records to upsert. A maximum of 200 records can be upsert in a batch.                    |
-| **variables** | **map[string,object]**                                                                      | The initial values for the variables the pipeline operates on and transforms throughout its steps. | [optional] |
+| Name          | Type                                                                                                            | Description                                                                                        | Notes      |
+| ------------- | --------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- | ---------- |
+| **pipeline**  | [**\Sajari\Model\BatchUpsertRecordsRequestPipeline**](BatchUpsertRecordsRequestPipeline.md)                     |                                                                                                    | [optional] |
+| **records**   | **object[]**                                                                                                    | A list of records to upsert. A maximum of 200 records can be upserted in a batch.                  |
+| **variables** | [**map[string,AnyOfNullBooleanIntegerLongFloatDoubleString]**](AnyOfNullBooleanIntegerLongFloatDoubleString.md) | The initial values for the variables the pipeline operates on and transforms throughout its steps. | [optional] |
```

Comment:
Is `AnyOfNullBooleanIntegerLongFloatDoubleString` a valid type? Also noting that this file `AnyOfNullBooleanIntegerLongFloatDoubleString.md` doesn't seem to exist